support-frontend/assets/components/productPage/productPageHero/productPageHero.jsx

```diff
@@ -72,41 +77,38 @@ const HeroHeading = ({
   </div>
 );
 
-const PaymentSelectionContainer = () => (
-  <div className="payment-selection-container">
-    <LeftMarginSection>
-      <PaymentSelection />
-    </LeftMarginSection>
-  </div>
-);
-
 const ProductPageHero = ({
-  overheading, heading, content, modifierClasses, children, appearance, hasCampaign, dailyEditionsVariant,
+  modifierClasses, children, appearance, showProductPageHeroHeader, ...props
 }: PropTypes) => (
   <header>
     <HeroWrapper {...{ modifierClasses, appearance }}>
       {children}
     </HeroWrapper>
-
-    {dailyEditionsVariant ? (
-      <PaymentSelectionContainer />
-      ) : (
-        <div>
-          <HeroHeading {...{ hasCampaign }}>
-            <HeadingBlock overheading={overheading} >{heading}</HeadingBlock>
-          </HeroHeading>
-          {content && <HeroHanger>{content}</HeroHanger>}
-        </div>
-      )}
+    {showProductPageHeroHeader && <ProductPageHeroHeader {...props} />}
   </header>
 );
 
+const ProductPageHeroHeader = ({
+  overheading, heading, content, hasCampaign,
+}: ProductPageHeroHeaderTypes) => (
+  <div>
+    <HeroHeading {...{ hasCampaign }}>
+      <HeadingBlock overheading={overheading} >{heading}</HeadingBlock>
+    </HeroHeading>
+    {content && <HeroHanger>{content}</HeroHanger>}
+  </div>
+);
+
 ProductPageHero.defaultProps = {
   children: null,
   content: null,
   ...HeroWrapper.defaultProps,
+  showProductPageHeroHeader: false,
```

Comment:
I was thinking that if you set the default to true, then you could leave it out everywhere except Digipack, but happy to merge as is.